tile_geotiff.py

```diff
@@ -18,13 +18,18 @@
 import random
 import re
 from argparse import ArgumentParser
-from typing import Tuple
+from typing import Any, List, Tuple
+
+from matplotlib.widgets import Button
```

Comment:
Just noticed that this is not inside of the `try...except` block. What you should do is move this in there and instead of just `pass`ing on import error, you should define `Button = None` so that it can still be used in the type annotations